docs/docs/usage/sbt-projects.md

```diff
@@ -3,4 +3,8 @@ layout: doc-page
 title: "Using Dotty with sbt"
 ---
 
-See the [example project](https://github.com/lampepfl/dotty-example-project).
+The fastest way to have a new project in dotty is using one of the following templates
+*  [Dotty Template Project](https://github.com/lampepfl/dotty.g8)
+*  [Dotty Template Project (cross compiling with Scala 2)](https://github.com/lampepfl/dotty-cross.g8)
+
+For documentation see the [example project](https://github.com/lampepfl/dotty-example-project).
```

Comment:
Could we have this on the main page? I'd put this in the `_includes` dir and import it in both locations, so as not to duplicate + `sbt new lampepfl/dotty.g8`